src/test/README.md

```diff
@@ -33,19 +33,31 @@ the `src/qt/test/test_main.cpp` file.
 
 ### Running individual tests
 
-`test_bitcoin` has some built-in command-line arguments; for
-example, to run just the `getarg_tests` verbosely:
+`test_bitcoin` accepts the command line arguments from the boost framework.
+For example, to run just the `getarg_tests` suite of tests:
 
-    test_bitcoin --log_level=all --run_test=getarg_tests -- DEBUG_LOG_OUT
```

Comment:
I don't think `--printtoconsole=1` is a replacement for `-- DEBUG_LOG_OUT`. `DEBUG_LOG_OUT` would work even if the unit test didn't start a Bitcoin Core node internally, no?

Reply:
Right. However, currently all unit tests create such a node: `BOOST_FIXTURE_TEST_SUITE(..., X)` where `X` is `BasicTestingSetup` or something that has inherited from it. So I think `DEBUG_LOG_OUT` can be completely phased out, following this PR.

Reply:
It is possible to create "pure" unit tests without any fixture, IIRC

Reply:
Ok, are you suggesting to restore the mention of `DEBUG_LOG_OUT` in `src/test/README.md` or to not phase out `DEBUG_LOG_OUT` after this PR?

Reply:
I guess it is fine to remove `DEBUG_LOG_OUT`. I just wanted to mention that not all test spin up a node.